std/assembly/math.ts

```diff
@@ -1880,6 +1880,46 @@ function tan_kernf(x: f64, odd: i32): f32 { // see: musl/tree/src/math/__tandf.c
   return <f32>(odd ? -1 / r : r);
 }
 
+// @ts-ignore: decorator
+@inline
+function log2f(x: f64): f64 {
+  const
```

Comment:
Perhaps reference `metallic/src/math/float/log2f.c`.

Reply:
And this seems to be rather `metallic/math/float/finite/log2f.h`. Quite confusing structure over there.